src/DiagnosticsDialog.cpp

```diff
@@ -26,7 +26,13 @@
 #include <QClipboard>
 
 #include <iostream>
+#if defined(_WIN32)
+#include <io.h>
+#include <windows.h>
+#define usleep Sleep
```

Comment:
please use the Qt API instead to sleep

Reply:
@mpreisler is that a note to yourself? https://github.com/OpenSCAP/scap-workbench/commit/4bbc842009d64f627c2f72f59ad553594a4ee41a ;)